src/app/rundown-view/components/offsetable-piece/offsetable-piece.component.ts

```diff
@@ -84,6 +85,11 @@ export class OffsetablePieceComponent {
 
   @HostBinding('class')
   public get getPieceTypeModifierClass(): string {
-    return (this.piece as Tv2Piece).metadata.type.toLowerCase().replace(/_/g, '-')
+    const piece: Tv2Piece = this.piece as Tv2Piece
+    const pieceTypeModifierClass: string = piece.metadata.type.toLowerCase().replace(/_/g, '-')
+    if (piece.metadata.audioMode && piece.metadata.audioMode === Tv2AudioMode.VOICE_OVER) {
+      return pieceTypeModifierClass + '-voice-over'
+    }
+    return pieceTypeModifierClass
```

Comment:
If we use two classes instead of one, we can also simplify this a bit to something like:
```typescript
return [
  piece.metadata.type.toLowerCase().replaceAll('_', '-'),
  (piece.metadata.audioMode ?? Tv2AudioMode.FULL).toLowerCase().replace('_', '-'),
].join(' ')

Reply:
Just rember to use `.replaceAll` in all instances where replacing is done, when using strings as patterns, or continue to use a regex as has already been used.

Personally i'd want to use `.replaceAll` in these very simple replace cases.

If more places than this are converting a Screaming Snake case string to a css class name, then maybe a helper method could be good?

Reply:
Until we change to es2021 then replaceAll is not an option. It will not be used for now.

Reply:
Implemented!